### PR TITLE
Documentation for business cases

### PR DESCRIPTION
**Pull request by blackakula**

### CODE REVIEW

File: doc/basics.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ This is the highest-order function in Basics.
 ### Read more
 - [README home](../readme.md)
 - [Order model](./sales.pdf)
+- [Business scenarios](./doc/sales/business-scenarios.pdf)
```
Comment (andrew-demb) [inline doc/basics.md:148]:
```suggestion
- [Business scenarios](./sales/business-scenarios.pdf)
```

Same for other files under `doc/`

Comment (blackakula) [reply doc/basics.md:148]: It is part of [the documentation](https://github.com/chantelle-lingerie/sales/blob/c40c51e3402edec83e205f84dd2e2b22fd2bf982/doc/sales/business-scenarios.pdf).
Don't want to introduce a new folder in the root.

Comment (andrew-demb) [reply doc/basics.md:148]: This file already in docs directory, link should omit `/doc/` because instead it leads to `doc/doc/sales/business-cases.pdf`

Comment (andrew-demb) [reply doc/basics.md:148]: You can check generated link here - https://www.github.com/chantelle-lingerie/sales/tree/doc-business-cases/doc%2Fbasics.md

Comment (blackakula) [reply doc/basics.md]: Done. Thank you!

**Comment by blackakula** [comment]
TODO:
- [ ] No need to be [PDF](https://github.com/chantelle-lingerie/sales/blob/doc-business-cases/doc/sales/business-scenarios.pdf) - re-write as markdown: one common + for each test scenario
- [ ] Explain items IDs (different or same item) in calculations

**Review by aliankarali — APPROVED** [review]

**Pull request merged**